src/pad/src/RDLRouter.cpp

```diff
@@ -1152,6 +1152,8 @@ std::vector<RDLRouter::TargetPair> RDLRouter::generateRoutingPairs(
   odb::dbTechLayer* bump_pin_layer = get_other_layer(bump_accessvia_);
   odb::dbTechLayer* pad_pin_layer = get_other_layer(pad_accessvia_);
 
+  const double dbus = block_->getDbUnitsPerMicron();
```

Comment:
This looks to be defined far from it's use

Reply:
I can move it down closer.